src/ngx-slider/lib/slider.component.ts

```diff
@@ -866,8 +948,9 @@ export class SliderComponent implements OnInit, AfterViewInit, OnChanges, OnDest
       this.manageEventsBindings();
     }
     this.updateDisabledState();
+    this.updateAriaLabel();
     this.calculateViewDimensions();
-    this.refocusPointerIfNeeded();
+    // this.refocusPointerIfNeeded();
```

Comment:
What's the reason for commenting out this?

Reply:
It was causing an issue (i dont remember what was it), but we could try uncoment and test it again.